pharus/server.py

```diff
@@ -64,7 +66,9 @@ def wrapper(**kwargs):
                 environ["PHARUS_PUBLIC_KEY"],
                 algorithms="RS256",
             )
-            return function(jwt_payload, **kwargs)
+            jwt_encoded = request.headers.get("Authorization").split()[1]
```

Comment:
Surprised that the bearer token you got from Keycloak got this far. I would have thought it would fail the `decode` step since it was not signed by Pharus. Let's review this approach together.

Think it might be better to rename `jwt_payload` to `connect_creds`. That way it is a bit more descriptive of the actual content. That way, we could reuse this in an OIDC setup too.

This would technically be a breaking change for some edge cases with `override` and if someone is using Pharus' Python API directly but I think it is worth it. Feel like we are the primary users of this so we probably know of the cases where it will be affected.